doc/tutorials/using_windows_as_user_vm.rst

```diff
@@ -105,7 +105,7 @@ Prepare the Script to Create an Image
    mem_size=4096M
    acrn-dm -A -m $mem_size -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio \
      -s 2,passthru,0/2/0 \
-     -s 8,virtio-net,tap0 \
+     -s 8,virtio-net,tap=0 \
```

Comment:
```suggestion
     -s 8,virtio-net,tap=tap0 \
```

Reply:
As previous.
Could we use tap=LaaG to show the virtio-net format?

Reply:
The "dm: remove the assume of virtio-net device name"  was PR now.
The new format is tap=tap0, so we can resolved this.